app/TestTCP/testenv.hpp

```diff
@@ -183,6 +182,17 @@ class TestEnv2 : public ::testing::Test
 		host2->getPort(1)->setPropagationDelay(TimeUtil::makeTime(1, TimeUtil::MSEC));
 
 		switchingHub = new Switch("Switch1", &netSystem, Unreliable);
+
+		/*
+		 *  In order to make random components controllable.
+		 *  Editor : payload@kaist.ac.kr
+		 */
```

Comment:
We will appreciate you in contributor list.
https://github.com/ANLAB-KAIST/KENSv3/graphs/contributors

Reply:
Okay, I'll remove those 4 lines and make a commit